res/css/views/rooms/_EventTile.scss

```diff
@@ -297,6 +297,8 @@ $left-gutter: 64px;
 
 /* De-zalgoing */
 .mx_EventTile_body {
+    text-align: start;
+    display: block;
```

Comment:
Rather than changing `mx_EventTile_body` elements (which are `span`s) to `display: block`, it would be more straightforward to instead make them `div`s and get this for free.

It would also make sense to me to change `mx_EventTile_content` elements to `div`s as well, and remove their `display: block` style, so we are consistently using `div` vs. `span` for their intended meaning.

Reply:
Okay I've changed them over to `div`s. This mostly works except it breaks the layout for edited messages and `/me` messages. I'm not sure how to fix this.

<img width="378" alt="screenshots" src="https://user-images.githubusercontent.com/5855073/100690355-f4597800-334b-11eb-9349-8f9e1e63882d.png">

Reply:
Ah, hmm... 😖 Okay, I'll have to think about how we'd want to handle the edited case.

Reply:
what is the issue with edited case?

Reply:
See the screenshot above. The `(edited)` label is in the wrong place.

Reply:
why not set its display to block and then text-align to start?